docs-next/components/Page.tsx

```diff
@@ -13,110 +14,119 @@ import { TableOfContents } from './TableOfContents';
 
 export const Page = ({
   headings = [],
+  title,
   children,
   isProse,
 }: {
   headings?: Heading[];
   children: ReactNode;
+  title?: string;
   isProse?: boolean;
 }) => {
   const [mobileNavCollapsed, setMobileNavCollapsed] = useState(true);
   const contentRef = useRef<HTMLDivElement | null>(null);
+  const metaTitle = title ? `Keystone Next Documentation - ${title}` : `Keystone Next`;
   return (
-    <div className="pb-24">
-      <header className="py-4 border-b border-gray-200">
-        <div className="w-full max-w-5xl mx-auto flex items-center justify-between sticky px-2">
-          <h2 className="flex items-center">
-            <img
-              alt="KeystoneJS Logo"
-              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIwIiBoZWlnaHQ9IjIyMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZGVmcz4KICAgIDxsaW5lYXJHcmFkaWVudCB4MT0iMCUiIHkxPSIwJSIgeDI9IjUwJSIgeTI9IjcxLjkyMSUiIGlkPSJsb2dvLXN2Zy1ncmFkaWVudCI+CiAgICAgIDxzdG9wIHN0b3AtY29sb3I9IiM1QUU4RkEiIG9mZnNldD0iMCUiLz4KICAgICAgPHN0b3Agc3RvcC1jb2xvcj0iIzI2ODRGRiIgb2Zmc2V0PSIxMDAlIi8+CiAgICA8L2xpbmVhckdyYWRpZW50PgogIDwvZGVmcz4KICA8cGF0aCBkPSJNMjkwLjEzNiA0N0g0MDcuNThjMTcuODMgMCAyNC4yOTcgMS44NTcgMzAuODE1IDUuMzQzIDYuNTE5IDMuNDg2IDExLjYzNCA4LjYwMiAxNS4xMiAxNS4xMiAzLjQ4NyA2LjUxOSA1LjM0MyAxMi45ODQgNS4zNDMgMzAuODE1djExNy40NDRjMCAxNy44My0xLjg1NiAyNC4yOTYtNS4zNDMgMzAuODE1LTMuNDg2IDYuNTE4LTguNjAxIDExLjYzNC0xNS4xMiAxNS4xMi02LjUxOCAzLjQ4Ni0xMi45ODQgNS4zNDMtMzAuODE1IDUuMzQzSDI5MC4xMzZjLTE3LjgzIDAtMjQuMjk2LTEuODU3LTMwLjgxNS01LjM0My02LjUxOC0zLjQ4Ni0xMS42MzQtOC42MDItMTUuMTItMTUuMTItMy40ODYtNi41MTktNS4zNDMtMTIuOTg0LTUuMzQzLTMwLjgxNVY5OC4yNzhjMC0xNy44MyAxLjg1Ny0yNC4yOTYgNS4zNDMtMzAuODE1IDMuNDg2LTYuNTE4IDguNjAyLTExLjYzNCAxNS4xMi0xNS4xMkMyNjUuODQgNDguODU3IDI3Mi4zMDUgNDcgMjkwLjEzNiA0N3ptMTEuNzYyIDU2Ljc2VjIxOGgyNS4xMnYtMzYuOGwxNC40LTE0LjU2IDM0LjQgNTEuMzZoMzEuNTJsLTQ4Ljk2LTY5LjEyIDQ0LjY0LTQ1LjEyaC0zMS4zNmwtNDQuNjQgNDcuMzZ2LTQ3LjM2aC0yNS4xMnoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0yMzguODU4IC00NykiIGZpbGw9InVybCgjbG9nby1zdmctZ3JhZGllbnQpIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz4KPC9zdmc+Cg=="
-              className="w-8 h-8 mr-2"
-            />
-            <Link href="/" passHref>
-              <a className="bg-clip-text text-transparent hover:text-transparent bg-gradient-to-r from-lightblue-500 to-indigo-600 hover:from-lightblue-400 hover:to-indigo-700 text-xl font-semibold">
-                Keystone Next
+    <Fragment>
+      <Head>
+        <meta property="og:site_name" content={metaTitle} />
```

Comment:
Pro tip: If you throw a `key` on this, it'll allow other components further down the tree to override the value without causing a duplicate tag to appear (as long as the `key`s match)

Similarly for the `<title>` below, but Next.js special-cases `<title>` already for you, so no need to manually add a `key` there.